specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/examples/AzureLargeInstance_Delete.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "api-version": "2023-07-20-preview",
+    "subscriptionId": "f0f4887f-d13c-4943-a8ba-d7da28d2a3fd",
+    "resourceGroupName": "myResourceGroup",
+    "azureLargeInstanceName": "myAzureLargeInstance"
+  },
+  "responses": {
+    "200": {},
+    "204": {},
+    "501": {}
```

Comment:
>  "501": {} [](http://example.com/codeflow?start=2&length=11)

don't need to include example 501 responses

Reply:
I removed the 501 responses from the examples but this now fails model validation